### ISSUE TITLE
Obsolete zway switch update command

### ISSUE THREAD

**Issue by ds2216** [MMore/pimatic-plugin-template#8]
The plugin sends update commands for z-wave switches to the z-way server every (@config.interval) seconds. That forces the server to communicate with the z-wave switch in the configured intervals. Z-Way Server is responsible to keep track of the status of devices and if everything is configured well, sending an update command is not necessary. Line 76 in z-way.coffee is obsolete. Please consider changing this in the next update.

**Comment by MMore** [comment]
Hey @ds2216, thanks for this information. But you can set the interval to 0 to avoid this polling. We implemented polling to ensure that after another application changed the device state it is represented correctly in pimatic.
Maybe I misunderstood you...

**Comment by ds2216** [comment]
Polling is ok and we really need it to know when a device state was changed manually or by other applications. But the @getState() in line 77 does the job well enough. 
Z-Wave devices always send state changes to the controller (RazBerry). So the  the z-way server always has the correct state and it does not need to the ask the device periodically. 
Believe me, I commented line 76 out 4 weeks ago and it's working well. I do switch my switches manually too and pimatic always has the correct state ;-).
